src/frontend/Routes/Routing.jsx

```diff
@@ -25,6 +29,11 @@ export const Routing = () => {
       <Route path="/products/:productId" element={<Product />} />
       <Route path="/checkout" element={<Checkout />} />
       <Route path="/address" element={<Address />} />
+      <Route path="/profile" element={<Profile />}>
+        <Route index element={<UserProfile />} />
+        <Route path="addressDetails" element={<AddressDetails />} />
```

Comment:
/ is missing in both the pathnames

Reply:
for nested routes we do not need to give /